specification/app/resource-manager/Microsoft.App/stable/2024-03-01/ManagedEnvironments.json

```diff
@@ -1259,6 +1259,22 @@
                   "$ref": "#/definitions/Mtls"
                 }
               }
+            },
+            "peerTrafficConfiguration": {
+              "description": "Peer traffic settings for the Managed Environment",
+              "type": "object",
+              "properties": {
+                "encryption": {
+                  "description": "Peer traffic encryption settings for the Managed Environment",
+                  "type": "object",
+                  "properties": {
+                    "enabled": {
+                      "type": "boolean",
```

Comment:
>boolean [](http://example.com/codeflow?start=31&length=7)

Consider making this an Enum. Also please check the other Lint Diff warnings for this PR here : 

[https://github.com/Azure/azure-rest-api-specs/pull/28159/checks?check_run_id=22421639508](https://github.com/Azure/azure-rest-api-specs/pull/28159/checks?check_run_id=22421639508)

Reply:
Thanks for your feedback. It's intended to keep consistent with the existing `peerAuthentication.mtls.enabled` which is also boolean. For the rest "AvoidNestedProperties" warnings, it isn't applicable here.